content/en/containers/troubleshooting/_index.md

```diff
@@ -2,8 +2,171 @@
 title: Container Troubleshooting
 kind: documentation
 description: Troubleshooting containers-related issues
+further_reading:
+- link: "/containers/troubleshooting/duplicate_hosts"
+  tag: "Documentation"
+  text: "Duplicate hosts with Kubernetes on AWS (EC2 or EKS)"
 ---
 
-{{< whatsnext desc="This section includes the following topics:">}}
-  {{< nextlink href="/containers/troubleshooting/duplicate_hosts">}}<u>Duplicate hosts</u>: If you are running the Datadog Agent in a Kubernetes environment on AWS (fully self-managed on EC2, or using EKS) you may see a problem with duplicate hosts.{{< /nextlink >}}
-{{< /whatsnext >}}
\ No newline at end of file
+This page provides troubleshooting information for container monitoring.
+
+
+There are three methods of deploying the Agent:
+1. As a [**container in a runtime**][1] 
+
+2. In a **cloud environment**, such as [Amazon ECS][2], [Fargate in an Amazon ECS environment][3], or [Amazon EKS][4]
+
+3. In a [Kubernetes environment][2]
+
+These different methods present unique deployment challenges. Use this page as a starting point to resolve issues. If you continue to have trouble, reach out to [Datadog support][6] for further assistance. 
+
+For details on Agent release updates or changes, refer to Datadog’s [release notes][7].
+
+## General issues
+
+### Environment variables are not being set, and tags are not injected
+
+A useful way to inject [environment variables][8] or to configure a DogStatsD library is to implement the [Admission Controller][9] feature on the Cluster Agent. **Note**: The Cluster Agent must be deployed and running _before_ the application is deployed.
+
+### Metrics are not appearing on the Datadog Web Platform
+
+Verify that the following are true:
+
+- The metrics endpoint is exposed and is open for the the Agent to reach.
+
+- There are no proxies or firewalls that might impede the Agent from accessing the endpoint. 
+
+- Agent has [Autodiscovery][10] enabled.
+
+
+### Logs are not collected
+
+There are two [environment variables][8] that can impact if logs will be collected and from which containers:
```

Comment:
did you mean the environment variables can...

have an impact? be impacted? impact something?

Reply:
good call gonna replace it with "affect"